packages/components/src/Skirt/_docs/Skirt.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { Text } from "~components/Text"
+import { Skirt } from "../index"
+import { SkirtCard } from "../subcomponents/SkirtCard"
+
+const meta = {
+  title: "Components/Skirt",
+  component: Skirt,
+  args: {
+    children: (
+      <SkirtCard>
```

Comment:
Story looks weird without the `TitleBlock` will most likely add the `Titleblock` in here when its migrated.